### PR TITLE
Update maintenance-mode.md

### PR DESCRIPTION
**Pull request by adisht**
Changes according to 
https://redislabs.atlassian.net/browse/RED-28017

### CODE REVIEW

File: content/rs/clusters/maintenance-mode.md [hunk 1]
```diff
@@ -87,6 +88,9 @@ To turn maintenance mode off after you finish server maintenance, run:
 ```sh
 rladmin node <node_id> maintenance_mode off
 ```
+{{<note>}}
+“Maintenance mode off” can finish successfully even though it did not restore the snapshot.
```
Comment (adisht) [inline content/rs/clusters/maintenance-mode.md:92]: Hi @lanceleonard I need to change this into something else according to information from https://redislabs.atlassian.net/browse/RED-68917
The new text should say: 

Note that running “Maintenance mode off” will not have an effect if the correlating snapshot is missing.

File: content/rs/clusters/maintenance-mode.md [hunk 2]
```diff
@@ -156,7 +160,7 @@ After turning on maintenance mode for node 2, Redis Enterprise saves a snapshot
 
 Now node 2 has `0/0` shards because shards cannot migrate to it while it is in maintenance mode.
 
-## Toggle maintenance mode via API
+## Set maintenance mode via API
 
 You can also turn maintenance mode on or off via [REST API requests]({{<relref "/rs/references/rest-api">}}) to [<nobr>POST `/nodes/{node_uid}/actions/{action}`</nobr>]({{<relref "/rs/references/rest-api/requests/nodes/actions#post-node-action">}}).
```
Comment (adisht) [inline content/rs/clusters/maintenance-mode.md:166]: Another addition following the conversation in https://redislabs.atlassian.net/browse/RED-68917:

"Maintenance mode can be turned off without restoring the snapshot using the CLI commands:
rladmin tune node 1 max_listeners 100
rladmin tune node 1 quorum_only disabled"

**Comment by lanceleonard** [comment]
#hold-off

[PR #2238](https://github.com/RedisLabs/redislabs-docs/pull/2238) includes changes to address the same concerns, along with additional edits/changes.  If/when that PR approved, this PR should be closed.

**Comment by lanceleonard** [comment]
Closing this PR in favor of [PR #2238](https://github.com/RedisLabs/redislabs-docs/pull/2238), which has been merged.

**Pull request closed without merging**